apps/watch/src/components/ShareDialog/ShareDialog.spec.tsx

```diff
@@ -135,9 +133,12 @@ describe('ShareDialog', () => {
     })
 
     it('should share video to twitter', () => {
+      mockRouter.push(
+        '/watch/the-story-of-jesus-for-children.html/english.html?r=0'
+      )
       const facebookUrl = `https://twitter.com/intent/tweet?url=${
```

Comment:
```suggestion
      const twitterUrl = `https://twitter.com/intent/tweet?url=${
```
Change to twitterUrl